src/client/Microsoft.Identity.Client/Internal/Requests/OnBehalfOfRequest.cs

```diff
@@ -38,7 +38,7 @@ protected override async Task<AuthenticationResult> ExecuteAsync(CancellationTok
             }
 
             await ResolveAuthorityAsync().ConfigureAwait(false);
-            MsalAccessTokenCacheItem msalAccessTokenItem = null;
+            MsalAccessTokenCacheItem cachedAccessToken = null;
```

Comment:
nit: since you changed the msalAccessTokenItem to cachedAccessToken.

Does it make sense to also change 
MsalRefreshTokenCacheItem and msalIdTokenItem?